### DISCUSSION TITLE
Change the starting White color while a model loads in model viewer

### DISCUSSION THREAD

**Discussion by rianlukog** [google/model-viewer#2530] [Q&A] [answered]
How to change the color of the loading that displays along with the progress bar.
by default: White

**Comment by elalish** [comment, marked as answer]
https://modelviewer.dev/docs/#entrydocs-loading-css-posterColor, though I would highly recommend supplying a poster image instead. Seamless posters can be easily generated in our [editor](https://modelviewer.dev/editor/).

**Reply by rianlukog** [reply]
Yes from what I have tried adding black; does give a dark grey 

Thanks for your response

**Reply by elalish** [reply]
You may also want to hide the progress mask, which is probably what's shifting your color: https://modelviewer.dev/docs/#entrydocs-loading-parts-defaultProgressMask
